resources/lang/en/site.php

```diff
@@ -14,5 +14,6 @@
 	'selectLocale'  => 'Select one of the supported languages',
 	'contact'		=> 'Contact',
 	'contact-us'	=> 'Contact Us',
+    'places'        => 'Places',
```

Comment:
Looks like the whitespace is off here. Not sure if there's a space or two too many or if there is an issue with tabs vs. spaces, but you probably want to fix this :)

Reply:
Arrgh yeah tabs/white space always does it. Will clean it up. Thanks!